src/split-panel/__tests__/split-panel.test.tsx

```diff
@@ -277,4 +278,17 @@ describe('Split panel', () => {
       expect(sidePositionTileElement?.disabled).toBeTruthy();
     });
   });
+  describe('has proper aria properties', () => {
+    test('split panel content has correct role', () => {
+      const { wrapper } = renderSplitPanel({ contextProps: { position: 'side' } });
+      const sidePanelElem = wrapper.findByClassName(styles['drawer-content-side'])?.getElement();
+      expect(sidePanelElem).toHaveAttribute('role', 'region');
+    });
+
+    test('split panel is labelled by panel header', () => {
+      const { wrapper } = renderSplitPanel({ contextProps: { position: 'side' } });
+      const sidePanelElem = wrapper.findByClassName(styles['drawer-content-side'])?.getElement();
+      expect(sidePanelElem?.getAttribute('aria-labelledby')).toMatch('split-panel-header');
```

Comment:
Looks good 👍 
I think it would be better to have the test more strict:
```
const labelId = sidePanelElem.getAttribute('aria-labelledby');
expect(sidePanelElem.querySelector(`#${labelId}`)!.textContent).toBe('Split panel header');
```

Reply:
Done